.github/workflows/dotnet_nugets.yml

```diff
@@ -0,0 +1,172 @@
+# Changing this name will cause github.run_number to be reset and affect the
```

Comment:
rlclientlib has the same version number script as VW - located at `.scripts/version_number.py` - so we should be able to generate a consistently increasing semver number without reference to `github.run_number`

Reply:
This repo does not use tagged versions therefore that version script is not going to produce a version which can be picked up by nuget as something newer. The incrementing run number is part of the semver prerelease info and therefore doesnt participate in versioning